src/Orchard.Web/Modules/Orchard.Users/Views/EditorTemplates/Parts/Users.SecuritySettings.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model Orchard.Users.Models.SecuritySettingsPart
+
+<fieldset>
+    <legend>@T("Security settings")</legend>
+    <div>
+        <label>@T("Life span of authentication cookies")</label>
+        @Html.TextBoxFor(m => m.AuthCookieLifeSpan, new { @class = "text medium" })
+        @Html.ValidationMessageFor(m => m.AuthCookieLifeSpan)
+        <p class="hint">@T("The time after signing in after which authentication cookies will be invalidated, in standard TimeSpan format (dd.hh:mm:ss).")</p>
```

Comment:
Don't mention `TimeSpan`, just the format.

Reply:
For reference https://msdn.microsoft.com/library/1d3t3c61(v=vs.100).aspx

Reply:
I went to the URL, but I am not sure what I am looking for in relation with the TimeSpan